cmd/globals.go

```diff
@@ -187,6 +187,13 @@ var (
 	globalCacheExpiry = 90
 	// Max allowed disk cache percentage
 	globalCacheMaxUse = 100
+
+	// RPC V1 - Initial version
+	// RPC V2 - format.json XL version changed to 2
+	// RPC V3 - format.json XL version changed to 3
+	// Current RPC version
+	globalRPCAPIVersion = RPCVersion{3, 0, 0}
```

Comment:
Is there a reason this change is in this commit? - this value is already present in master `globalRPCAPIVersion`

Reply:
This seems to have come up while rebasing onto master. Should I remove this?